SPECS/tcl/tcl.spec

```diff
@@ -31,6 +32,7 @@ Headers and development libraries for tcl
 
 %prep
 %setup -q -n %{name}%{version}
+%patch0 -p1
```

Comment:
How about we use `%autosetup` :)
```suggestion
%autosetup -n %{name}%{version} -p1
```

Reply:
updated for suggestions in commit below.